modules/lambda-adapter/src/index.ts

```diff
@@ -0,0 +1,70 @@
+import { ServerLogic } from "@phenyl/http-rules";
+
+import {
+  HttpMethod,
+  QueryStringParams,
+  ServerParams
+} from "@phenyl/interfaces";
+
+export type LambdaCallback = (err: any, result: any) => void;
```

Comment:
there is `Error` type in `@types/node`, should we use it?
`(err: Error | null | undefined, result: any) => void;`

Reply:
ooh, I missed that. Thanks!
I've modified the types to `(err?: Error | null, result?: any)`